crates/holochain_sqlite/tests/db_wrapper.rs

```diff
@@ -92,6 +94,7 @@ fn pool_size_is_limited() {
     assert_eq!(3, failed_count);
 }
 
+// TODO this test can be removed once `conn_permit` is removed from the public interface
```

Comment:
leftover or reserved for future?

Reply:
Mentioned in the PR description, this requires refactoring the holochain_cascade crate to not hold permits before I can action this TODO item